src/utils.js

```diff
@@ -0,0 +1,24 @@
+const core = require('@actions/core');
+const github = require('@actions/github');
+const { logInfo } = require('./log');
+
+async function createIssue(token, issueTitle, issueBody) {
+  try {
+    const octokit = github.getOctokit(token);
+
+    logInfo(issueBody);
```

Comment:
this doesn't look like something to be logged with level info. do we need to log this in the first place?